src/runmode-dpdk.c

```diff
@@ -1191,6 +1208,19 @@ static void PortConfSetChsumOffload(const DPDKIfaceConfig *iconf,
     }
 }
 
+static void PortConfSetVlanOffload(const DPDKIfaceConfig *iconf,
+        const struct rte_eth_dev_info *dev_info, struct rte_eth_conf *port_conf)
+{
+    if (iconf->vlan_strip_enabled) {
+        SCLogConfig("%s: hardware vlan stripping enabled", iconf->iface);
```

Comment:
I would only print this in case I actually set the VLAN offload (So a line below L1217)